gef.py

```diff
@@ -10590,9 +10590,6 @@ def parse_procfs_maps(cls) -> Generator[Section, None, None]:
     @classmethod
     def parse_gdb_info_proc_maps(cls) -> Generator[Section, None, None]:
         """Get the memory mapping from GDB's command `maintenance info sections` (limited info)."""
-        if GDB_VERSION < (11, 0):
```

Comment:
I don't think that's what you meant to do: on older versions, `Perms` is not exposed at all by GDB even on live process, for which we can know the permissions from procfs

However, removing this - and because `parse_gdb_info_proc_maps` is the first to be invoked when parsing mem mapping - it'll end up giving `rwx` to *everything*, which the unittest failures show quickly. 

![image](https://github.com/hugsy/gef/assets/590234/605bf486-5920-49a8-83d9-0c820d8cdea9)

The reason it was mocked for version higher than 11 was to restore support for coredumps where this info is missing.

Reply:
makes sense